tests/cli/analysis/rna/test_cli_analysis_rna.py

```diff
@@ -0,0 +1,55 @@
+""" Test the CLI for analysis rna """
+import logging
+
+from cg.cli.miprdrna import start
+from cg.apps.mip import MipAPI
+
+
+def test_start_dry(cli_runner, tb_api, mock_store, caplog):
+    """Test starting MIP"""
+    # GIVEN a cli function
+
+    context = {}
+    context['db'] = mock_store
+    context['tb_api'] = tb_api
+    context['rna_api'] = MipAPI('${HOME}/bin/mip', 'analyse rd_rna')
+    context['mip-rd-rna'] = {'mip_config': 'config.yaml'}
+
+    # WHEN we start a case in dry run
+    caplog.set_level(logging.INFO)
+    cli_runner.invoke(start,
+                      ['--dry', '--email', 'james.holden@scilifelab.se', 'angrybird'],
+                      obj=context)
+
+    # THEN the command should be printed
+    with caplog.at_level(logging.INFO):
+        assert '${HOME}/bin/mip analyse rd_rna angrybird --config_file config.yaml ' \
+               '--email james.holden@scilifelab.se --dry_run_all' in caplog.text
+
+
+def test_start(cli_runner, tb_api, mock_store, caplog, monkeypatch):
+    """Test starting MIP"""
+
+    # pylint: disable=unused-argument
```

Comment:
I'm no fan of silencing the linter, could we delete the arguments instead, e.g. "del kwargs"

Reply:
sure!